card/src/main/java/com/adyen/checkout/card/CardValidationUtils.kt

```diff
@@ -108,6 +115,24 @@ object CardValidationUtils {
         return FieldState(normalizedSecurityCode, validation)
     }
 
+    fun validateKcpBirthDateOrTaxNumber(birthDateOrTaxNumber: String): FieldState<String> {
+        val inputLength = birthDateOrTaxNumber.length
+        val validation = when {
+            inputLength == KCP_BIRTH_DATE_LENGTH && DateUtils.matchesFormat(birthDateOrTaxNumber, KCP_BIRTH_DATE_FORMAT) -> Validation.Valid
```

Comment:
Did you check this validation with the other platforms? `yyMMdd` seems odd to me, just wanted to double check.

Reply:
Yeah here you can find it's mentioned as well https://hub.is.adyen.com/our-solution/products-and-features/payment-methods/korean_cards_direct_api/#api-request-checkout-api-classic-api